### PR TITLE
Migrate to swift-bundler

### PR DESCRIPTION
**Pull request by nekename**
This PR migrates Texture Viewer to stackotter's [swift-bundler](https://swiftbundler.dev).

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// swift-tools-version:5.5
+
+import PackageDescription
+
+let package = Package(
+    name: "TextureViewer",
```
Comment (stackotter) [inline Package.swift]: Make the indentation consistently two spaces

File: Package.swift [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    ],
+    targets: [
+        .executableTarget(
+            name: "TextureViewer",
+            dependencies: [
+              "DeltaLogger"
+            ],
+            resources: [
+              .process("Metal/")
+            ]),
```
Comment (stackotter) [inline Package.swift]: Pop the last bracket onto its own line

**Review by stackotter — CHANGES_REQUESTED** [review]
Looks mostly good, just a few minor formatting changes. Also, I think you shouldn't need the `main.swift` file and you can just keep the `@main` attribute in TextureViewerApp.swift, but I'm assuming you ran into some issue that caused you to do that?

**Comment by nekename** [comment]
> but I'm assuming you ran into some issue that caused you to do that?

Nah, I just saw that swift bundler generated it, and that Delta Client had a main.swift, so moved it to be like that. I'll revert it.

**Review by stackotter — APPROVED** [review]
Thanks for migrating this to Swift Bundler

**Pull request merged**